test/extended/default_builds_test.go

```diff
@@ -0,0 +1,80 @@
+// +build default
+
+package extended
+
+import (
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	. "github.com/GoogleCloudPlatform/kubernetes/test/e2e"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = Describe("STI build with .sti/environment file", func() {
+	defer GinkgoRecover()
+
+	var imageStreamFixture = filepath.Join("..", "integration", "fixtures", "test-image-stream.json")
+	var stiEnvBuildFixture = filepath.Join("fixtures", "test-env-build.json")
+	var oc = exutil.NewGinkoCLI("build-sti-env", kubeConfigPath())
```

Comment:
Thinking on it, you should actually login in as a non-privileged user, request a new user, and then make certain that the default rights created in the new project are sufficient to run this test.  If they aren't, we should either fix the defaults or describe in this test the code required to get those rights.